.github/workflows/ci.yaml

```
@@ -162,6 +164,16 @@ jobs:
         if: always() && matrix.env.CCOV && steps.ici.outputs.target_test_results == '0'
         with:
           files: ${{ env.BASEDIR }}/target_ws/coverage.info
+      - name: Install sonar-scanner and build-wrapper
+        if: always() && matrix.env.CCOV && steps.ici.outputs.target_test_results == '0'
+        uses: SonarSource/sonarcloud-github-c-cpp@v1
+      - name: Run sonar-scanner
+        if: always() && matrix.env.CCOV && steps.ici.outputs.target_test_results == '0'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: sonar-scanner --define project.settings=src/moveit2/sonar-project.properties
```

I don't have write access here:
```suggestion
        run: sonar-scanner -X --define project.settings=src/moveit2/sonar-project.properties
```

adding `-X` should help us see what the inner error is here.